docs/installation.rst

```diff
@@ -70,3 +70,36 @@ Once your configuration is done, you can launch the deployment :
 .. code-block:: bash
 
     (venv) $ enos deploy
+
+
+The deployment is the combination of the following three phases:
+
+1. Acquire raw resources that are necessary for the deployment of
+   OpenStack. Enos acquires resources according to the ``provider``
+   and ``resources`` information in the reservation file. Someone can
+   perform this phase by calling ``enos up``.
+
+2. Deploy OpenStack on resources gotten during the previous phase.
+   Enos uses the list of resources provided by the previous phase and
+   combines them with information specified by the ``inventory`` key
+   to produce a file that gives a mapping of which OpenStack services
+   have to be deployed on which resources. Enos then calls the
+   Kolla-Ansible tool with this file to deploy, in a containerized
+   manner, OpenStack services on the right resources. Someone can
```

Comment:
"to deploy the containerized OpenStack services to the right resources."

Reply:
Done